setup.cfg

```
@@ -55,8 +55,8 @@ packages = find_namespace:
 python_requires = >=3.7
 include_package_data = True
 install_requires =
-    metomi-rose==2.1.*
-    cylc-flow==8.1.*
+    # metomi-rose==2.*.*
+    # cylc-flow==8.*.*
```

❓

---

Accidentally committed.
```suggestion
    metomi-rose==2.1.*
    cylc-flow==8.1.*
```